bin/rucio

```diff
@@ -2310,13 +2310,13 @@ if __name__ == '__main__':
 
     logger = None
```

Comment:
this module-wide variable is shadowed by the parameter "logger" and makes it hard to track usage of either in the review. I would suggest renaming this to something like `module_logger`